commit a77b15a60cb1d54d87e8794bfbc94c9ccee679ed
Author: Mikel Astiz <mikel.astiz@bmw-carit.de>
Date:   Fri Jun 28 10:56:27 2013 +0200

    Bluetooth: Add HCI authentication capabilities macros
    
    Add macros for the HCI capabilities as described in the Bluetooth Core
    Specification v4.0, Volume 2, part E, section 7.1.29.
    
    Signed-off-by: Mikel Astiz <mikel.astiz@bmw-carit.de>
    Signed-off-by: Gustavo Padovan <gustavo.padovan@collabora.co.uk>

diff --git a/include/net/bluetooth/hci.h b/include/net/bluetooth/hci.h
index 3c592cf473da..a01fbb4a0499 100644
--- a/include/net/bluetooth/hci.h
+++ b/include/net/bluetooth/hci.h
@@ -296,6 +296,12 @@ enum {
 #define HCI_AT_GENERAL_BONDING		0x04
 #define HCI_AT_GENERAL_BONDING_MITM	0x05
 
+/* I/O capabilities */
+#define HCI_IO_DISPLAY_ONLY	0x00
+#define HCI_IO_DISPLAY_YESNO	0x01
+#define HCI_IO_KEYBOARD_ONLY	0x02
+#define HCI_IO_NO_INPUT_OUTPUT	0x03
+
 /* Link Key types */
 #define HCI_LK_COMBINATION		0x00
 #define HCI_LK_LOCAL_UNIT		0x01